powdr_cli/src/main.rs

```diff
@@ -473,9 +474,14 @@ fn run_rust<F: FieldElement>(
     coprocessors: riscv::CoProcessors,
     just_execute: bool,
 ) -> Result<(), Vec<String>> {
-    let (asm_file_path, asm_contents) =
-        compile_rust(file_name, output_dir, force_overwrite, &coprocessors)
-            .ok_or_else(|| vec!["could not compile rust".to_string()])?;
+    let (asm_file_path, asm_contents) = compile_rust(
+        file_name,
+        output_dir,
+        force_overwrite,
+        &coprocessors,
+        just_execute,
```

Comment:
For now, we include a bootloader whenever the `--just-execute` flag is active.

Reply:
Or maybe we should just completely turn it off and add another `--continuations` flag to the main binary?

Reply:
Changed it.